network/connecteventmanager.go

```diff
@@ -186,7 +190,8 @@ func (c *connectEventManager) MarkUnresponsive(p peer.ID) {
 //
 // - When we're connected to the peer, this will mark the peer as responsive (from unresponsive).
 // - When not connected, we ignore this call. Unfortunately, a peer may disconnect before we process
-//   the "on message" event, so we can't treat this as evidence of a connection.
+//
+//	the "on message" event, so we can't treat this as evidence of a connection.
```

Comment:
```suggestion
//	the "on message" event, so we can't treat this as evidence of a connection.
```

Reply:
This is not up to go1.19 format rules.

If I do that go1.19 fmt do this:
```
//   - When we're connected to the peer, this will mark the peer as responsive (from unresponsive).
//   - When not connected, we ignore this call. Unfortunately, a peer may disconnect before we process
//     the "on message" event, so we can't treat this as evidence of a connection.
```
Let's not fight with fmt and keep this as-is.